crates/evm/src/evm/system_events.rs

```diff
@@ -45,6 +47,26 @@ fn system_event_to_transaction(event: SystemEvent, nonce: u64, chain_id: u64) ->
             max_fee_per_gas: u64::MAX as u128,
             ..Default::default()
         },
+        SystemEvent::BridgeInitialize(data) => TxEip1559 {
```

Comment:
should take the params contract uses, this will be set in the code level directly